source/game.programme.newsevent.bmx

```diff
@@ -366,6 +366,10 @@ Type TNewsEventCollection
 				'skip events already happened (and processed) or not
 				'happened at all (-> "-1")
 				If event.HasFlag(TVTNewsFlag.HAPPENING_PROCESSED) Or event.happenedTime = -1 Then Continue
+				'also ignore events which happened before game start 
+				'(eg fixed happen time news events)
+				If event.happenedTime < GetWorldTime().GetTimeStart() Then Continue
```

Comment:
Ich würde das Holen der Startzeit noch aus der Schleife rausziehen. Man könnte zudem überlegen, ob ein gewisser Offset sinnvoll wäre (oder überhaupt einen Unterschied machen würde - Spielstart 1985, Nachricht vom 28.12.1984 könnte noch verwendet werden).

Reply:
Du hast Recht, es koennte News mit "relativem Start" geben, die gezielt als "Altnews" linksseitig auftauchen sollen ("Neuer Sendestart steht bevor...") - wird derzeit aber nicht genutzt.

Reply:
OK. In dem Fall wird ja nur der Feldinhalt zurückgegeben und nicht wirklich was berechnet.